resources/js/Components/Table/EventTable/EventTable.tsx

```diff
@@ -1,18 +1,44 @@
-import {DataTable} from 'Components'
+import {Inertia} from '@inertiajs/inertia'
+import DoneIcon from '@mui/icons-material/Done'
+import Logout from '@mui/icons-material/Logout'
+import {Chip, IconButton, Typography} from '@mui/material'
+import {DataTable, Link} from 'Components'
+import dayjs from 'dayjs'
 import {useTranslation} from 'react-i18next'
 import {EventData} from 'Types/event-data'
+import route from 'ziggy-js'
 
 type EventTablePropType = {
     data: EventData[]
+    registered_only: boolean | null
 }
 
-const EventTable = ({data}: EventTablePropType) => {
+const EventTable = ({data, registered_only}: EventTablePropType) => {
```

Comment:
i assume by default if `registered_only` is null, it means it's false right?

then this case we can initialize `registered_only` to false, you can do this by 

`const EventTable = ({data, registered_only = false}: EventTablePropType) => {`

Reply:
Interesting, thank you